samples/mpsl/timeslot/src/main.c

```diff
@@ -81,10 +100,8 @@ static mpsl_timeslot_signal_return_param_t *mpsl_timeslot_callback(
 				MPSL_TIMESLOT_SIGNAL_ACTION_REQUEST;
 		} else {
 			/* No return action, so timeslot will be ended */
```

Comment:
What I meant in other comment is this line. This is no longer valid since the return action is specified.